src/main/resources/db/migration/common/V1_219__v48_without_transcription.sql

```diff
@@ -0,0 +1,17 @@
+-- V48 without transcription changes.
+ALTER TABLE court_case ALTER COLUMN case_number SET NOT NULL;
+ALTER TABLE court_case ALTER COLUMN case_closed SET NOT NULL;
+ALTER TABLE court_case ALTER COLUMN interpreter_used SET NOT NULL;
+
+ALTER TABLE node_register ALTER COLUMN device_type SET DEFAULT 'DAR';
```

Comment:
Move this into 218?

Reply:
This was done as part of v48 that's why is here.